financial_insights/streamlit/readme.md

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Maybe this readme file should exist in the higher level directory for financial assistant?

Reply:
Yes, thank you for noticing this.